package.json

```diff
@@ -21,7 +21,7 @@
   "homepage": "https://github.com/lennym/timeliner#readme",
   "dependencies": {
     "JSONStream": "^1.2.1",
-    "babar": "0.0.3",
+    "babar": "https://github.com/cubelets/babar#feature/configurable-viewport",
```

Comment:
Can you publish a namespaced npm package rather than use a github dependency please?

Github dependencies are not immutable, and can't be shrink-wrapped, so represent a potential security hole. I try to avoid ever normalising their use for public code.